tests/sqlite3.go

```diff
@@ -11,8 +11,9 @@ import (
 
 // Sqlite3URI returns a URI to spinup an in-memory Sqlite database.
 func Sqlite3URI(t *testing.T) string {
-	dbURI := "file::" + uuid.NewString() + ":?mode=memory&cache=shared&_foreign_keys=on"
+	dbURI := "file::" + uuid.NewString() + ":?mode=memory&cache=shared&_foreign_keys=on&_busy_timeout=5000"
```

Comment:
is the `busy_timeout` flag only relevant for the tests because we want to exit the long running test? or do we need the same flag in the usual flow too?

Reply:
We're using that flag in the [usual flow ](https://github.com/tablelandnetwork/go-tableland/blob/main/cmd/api/main.go#L67) too. I just added it here to make the tests db and the real with equal flags.